### PR TITLE
Migrate WSO2 Integrator and Broker with Analytics deployment to WSO2 EI version 6.2.0

### PR DESCRIPTION
**Pull request by chirangaalwis**

### CODE REVIEW

File: docker-compose/integrator-broker-analytics/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@
      Instead, extract the zip file and directly browse to `docker-ei-<released-version>/docker-compose/integrator-broker-analytics` folder. 
      
      > If you are to try out an already released tag, after executing 2nd step, checkout the relevant tag, 
-     i.e. for example: <br> git checkout tags/v6.1.1.4 and continue below steps.
+     i.e. for example: <br> git checkout tags/v6.2.0.4 and continue below steps.
```
Comment (msmshariq) [inline docker-compose/integrator-broker-analytics/README.md]: Shouldn't this be v6.2.0.1 ? Please fix all places.

File: docker-compose/integrator-broker-analytics/docker-compose.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ services:
       retries: 10
       start_period: 15s
   wso2ei-analytics:
-    image: docker.wso2.com/wso2ei-analytics:6.1.1
+    image: wso2ei-analytics:6.2.0
```
Comment (msmshariq) [inline docker-compose/integrator-broker-analytics/docker-compose.yml]: Why have we removed the docker.wso2.com prefix?

File: docker-compose/integrator-broker-analytics/docker-compose.yml [hunk 3]
```diff
@@ -86,7 +86,7 @@ services:
       - wso2ei-mysql
       - wso2ei-analytics
   wso2ei-integrator:
-    image: docker.wso2.com/wso2ei-integrator:6.1.1
+    image: wso2ei-integrator:6.2.0
```
Comment (msmshariq) [inline docker-compose/integrator-broker-analytics/docker-compose.yml]: Why is the docker.wso2.com prefix missing?

File: docker-compose/integrator-broker-analytics/docker-compose.yml (hunk 2 above)
Comment (chirangaalwis) [reply docker-compose/integrator-broker-analytics/docker-compose.yml]: @msmshariq will fix it. Couldn't revert the change after testing was finished.

File: docker-compose/integrator-broker-analytics/docker-compose.yml (hunk 3 above)
Comment (chirangaalwis) [reply docker-compose/integrator-broker-analytics/docker-compose.yml]: @msmshariq will fix it. Couldn't revert the change after testing was finished.

File: docker-compose/integrator-broker-analytics/README.md (hunk 1 above)
Comment (chirangaalwis) [reply docker-compose/integrator-broker-analytics/README.md]: +1

Comment (chirangaalwis) [reply docker-compose/integrator-broker-analytics/README.md]: For WSO2 Enterprise Integrator For Integration, Broker and Business Process (BPS) Use-cases With Analytics Support deployment, this fix would be available in https://github.com/wso2/docker-ei/pull/51.

**Pull request merged**